support-frontend/app/controllers/DigitalSubscription.scala

```diff
@@ -55,7 +56,15 @@ class DigitalSubscription(
     val description = stringsConfig.digitalPackLandingDescription
     val canonicalLink = Some(buildCanonicalDigitalSubscriptionLink("uk"))
     val september2019SalePromo = "DK0NT24WG"
-    val promoCodes = request.queryString.get("promoCode").map(_.toList).getOrElse(Nil) :+ september2019SalePromo
+    val annualIntroCodes: List[PromoCode] = List(
```

Comment:
This is the main point of this PR the rest is just removing some unused code and making sure that the correct promo code is passed from the product page to the checkout